fsspec/asyn.py

```diff
@@ -128,6 +172,7 @@ class AsyncFileSystem(AbstractFileSystem):
     # for _* methods and inferred for overridden methods.
 
     async_impl = True
+    disable_throttling = False
```

Comment:
Seems to me not a good instance- or class-wise setting (and it's not documented). Would prefer to set a global variable and/or config.

Reply:
The reason for implementations that do some other stuff in their `_put_file`/`_get_file` calls beside opening files on the local system.

Reply:
get and put should always use a local file. Indeed, all comms use sockets too, so there is reason to want to use batched gather everywhere. (In practice, the comms library like asyncio has a connection pool that does this throttling job too)